Automattic/jetpack
# VideoPress: Introduce `VideoQuickActions` component

Export out of `VideoRow` the `VideoQuickActions` section and reuse it across `VideoRow` and `VideoCard`.

#### Changes proposed in this Pull Request:

* Introduce `VideoQuickActions` component.
* Use it across `VideoRow` and `VideoCard`.

#### Other information:


#### Jetpack product discussion


p1662123350984669-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start Storybook
* Navigate to `Video Row`, `Video Card`, and `Video Quick Actions` stories.
* Confirm that quick actions appears and click works.

#### Demo


https://user-images.githubusercontent.com/1663717/188183696-3eb38832-547c-4144-ac49-abadd69e505a.mov
